gpsreceiver/gpsreceiver/antenna.py

```diff
@@ -85,7 +85,7 @@ class RtlSdrAntenna(Antenna):
     It's assumed that a single RTL-SDR is connected to the computer.
     """
 
-    def __init__(self, receiver: Receiver) -> None:
+    def __init__(self, receiver: Receiver, gain) -> None:
```

Comment:
Could you please add a type annotation here?

```suggestion
    def __init__(self, receiver: Receiver, gain: int) -> None:
```